### PR TITLE
Add datadog node template

### PR DESCRIPTION
**Pull request by thedavlee**
*Description of changes:*
- Added "Monitoring and Observability with Datadog" template for nodejs14

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: nodejs14.x/cookiecutter-aws-sam-hello-nodejs-datadog/.gitignore [hunk 1]
```diff
@@ -0,0 +1,229 @@
+
```
Comment (maxday) [inline nodejs14.x/cookiecutter-aws-sam-hello-nodejs-datadog/.gitignore:1]: I think we can clean this?

File: nodejs14.x/cookiecutter-aws-sam-hello-nodejs-datadog/{{cookiecutter.project_name}}/.gitignore [hunk 2]
```diff
@@ -0,0 +1,207 @@
+
```
Comment (maxday) [inline nodejs14.x/cookiecutter-aws-sam-hello-nodejs-datadog/{{cookiecutter.project_name}}/.gitignore:1]: same here

File: .gitignore [hunk 3]
```diff
@@ -3,3 +3,5 @@
 __pycache__
 .idea/
 venv/
+.gradle/
```
Comment (maxday) [inline .gitignore:6]: do we need to modify this main .gitignore file?

File: nodejs14.x/cookiecutter-aws-sam-hello-nodejs-datadog/{{cookiecutter.project_name}}/hello-world/tests/unit/test-handler.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+        expect(result).to.be.an('object');
+        // expect(result.statusCode).to.equal(200);
+        // expect(result.body).to.be.an('string');
+
+        // let response = JSON.parse(result.body);
+
+        // expect(response).to.be.an('object');
+        // expect(response.message).to.be.equal("Hello from serverless!");
+        // expect(response.location).to.be.an("string");
```
Comment (maxday) [inline nodejs14.x/cookiecutter-aws-sam-hello-nodejs-datadog/{{cookiecutter.project_name}}/hello-world/tests/unit/test-handler.js:24]: are those comments intentional?

File: manifest.json [hunk 5]
```diff
@@ -341,6 +341,15 @@
       "packageType": "Zip",
       "useCaseName": "Hello World Example"
     },
+    {
```
Comment (mndeveci) [inline manifest.json:344]: Can you remove changes to this file, as the `__comment__` suggest?

File: nodejs14.x/cookiecutter-aws-sam-hello-nodejs-datadog/{{cookiecutter.project_name}}/hello-world/package.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "hello_world",
+  "version": "1.0.0",
+  "description": "hello world sample for NodeJS",
+  "main": "app.js",
+  "repository": "https://github.com/awslabs/aws-sam-cli/tree/develop/samcli/local/init/templates/cookiecutter-aws-sam-hello-nodejs",
```
Comment (mndeveci) [inline nodejs14.x/cookiecutter-aws-sam-hello-nodejs-datadog/{{cookiecutter.project_name}}/hello-world/package.json:6]: nit: can you update the repository link here?

**Review by mndeveci — CHANGES_REQUESTED** [review]

File: manifest.json (hunk 5 above)
Comment (hawflau) [inline manifest.json:344]: Please remove the changes in this file from the PR. We have frozen this file and won't accept any change in this file.

**Review by hawflau — CHANGES_REQUESTED** [review]

**Comment by jfuss** [comment]
Closing given no response. Feel free to handle feedback and re-open or create a new PR.

**Pull request closed without merging**